srcpkgs/virt-manager/template

```diff
@@ -22,3 +20,29 @@ post_install() {
 	rm $DESTDIR/usr/share/glib-2.0/schemas/gschemas.compiled \
 		$DESTDIR/usr/share/icons/hicolor/icon-theme.cache
 }
+
+virt-manager-tools_package() {
+	noarch=yes
+	short_desc="Programs to create and clone virtual machines"
+	depends="libosinfo python3-gobject libvirt-python3 python3-urllib3 libxml2-python3 python3-requests"
+	pycompile_dirs="/usr/share/${pkgname}"
+	pycompile_version="$py3_ver"
+	pkg_install() {
+		vmove usr/bin/virt-clone
+		vmove usr/bin/virt-convert
+		vmove usr/bin/virt-install
+		vmove usr/bin/virt-xml
+		vmove usr/share/man/man1/virt-install.1
+		vmove usr/share/man/man1/virt-convert.1
+		vmove usr/share/man/man1/virt-clone.1
+		vmove usr/share/man/man1/virt-xml.1
+		vmove usr/share/virt-manager/virt-clone
+		vmove usr/share/virt-manager/virt-convert
+		vmove usr/share/virt-manager/virt-install
+		vmove usr/share/virt-manager/virt-xml
+		vmove usr/share/virt-manager/virtcli
+		vmove usr/share/virt-manager/virtconv
+		vmove usr/share/virt-manager/virtinst
+		vlicense COPYING
```

Comment:
`GPL-2.0-or-later` is on of the standard licenses, it does not require vlicense.

Reply:
Done